examples/rest-angular/tsconfig.app.json

```diff
@@ -0,0 +1,15 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
+{
+  "extends": "./tsconfig.base.json",
```

Comment:
Is it possible to remove TypeScript from this example?

The goal of examples is to have as little unrelated setup as possible, so it's easier for people to navigate through and understand what needs to be done in order to integrate MSW into their Angular project. If it's possible to omit TypeScript, that'd be awesome.

Reply:
Hi thanks for the review!
Angular comes with TypeScript so it wont be possible to remove TS from this example.

Reply:
Thanks for clarifying! Sorry, I don't have much experience with Angular and didn't know it comes with TS by default. That's awesome, though.